manifests/service.pp

```diff
@@ -0,0 +1,40 @@
+# Class: shibboleth_idp::service
+#
+# Manages the Jetty process
+#
+# Sample Usage:
+#
+#    sometype { 'foo':
+#      notify => Class['shibboleth_idp::service'],
+#    }
+#
+#
+class shibboleth_idp::service {
+  # The base class must be included first because parameter defaults depend on it
+  if ! defined(Class['shibboleth_idp::params']) {
+    fail('You must include the shibboleth_idp::params class before using any shibboleth_idp defined resources')
+  }
+  validate_bool($shibboleth_idp::params::service_enable)
+  validate_bool($shibboleth_idp::params::service_manage)
```

Comment:
If you need them then validate the params set in the init, not the pramas class.